src/Components/ApplicationsContainer.js

```diff
@@ -2,13 +2,21 @@ import React from 'react';
 import ApplicationCard from './ApplicationCard';
 import '../Styles/ApplicationContainer.scss';
 
-const ApplicationContainer = () => {
 
-  return(
+const ApplicationContainer = (apps) => {
+  let allApplications = apps.apps.applications.map(application => {
+    return (
+      <ApplicationCard
+          name={application.name}
+          email={application.email}
+          description={application.description}
+      />
+    )
+  })
+  let display = allApplications.length ? allApplications : <p className='no-applications-section'>No applications quite yet.</p>
```

Comment:
Good catch on the conditional. If there aren't any applications we don't want the user to be left with a blank page.